src/server/enrollment-system.js

```diff
@@ -362,6 +359,10 @@ function veteranToDependentFinancialsCollection(veteran) {
  * @returns {Object} ES system dependentFinancialsCollection message
  */
 function veteranToSpouseFinancials(veteran) {
+  if (!_.includes(['Married', 'Separated'], veteran.maritalStatus)) {
```

Comment:
`spouseFinancials` is the container for all spouse-related data, including name, address, city of birth, etc.. Returning `undefined` for anything other than Married/Separated lines up with the checks on the React & frontend validation side [like in this examples](https://github.com/department-of-veterans-affairs/healthcare-application/search?utf8=✓&q=%22this.props.data.maritalStatus.value+%3D%3D%3D+%27Married%27%22)